inc/itilsolution.class.php

```diff
@@ -366,10 +366,13 @@ function post_addItem() {
          }
       }
 
-      $this->item->update([
-         'id'     => $this->item->getID(),
-         'status' => $status
-      ]);
+      if (!isset($this->input['_linked_ticket'])) {
```

Comment:
This `if` instruction should encapsulate lines 352 to 368 too.

Reply:
done